app/controllers/api/edge.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use Ahc\Jwt\JWT;
+use Ahc\Jwt\JWTException;
+use Appwrite\Extend\Exception;
+use Appwrite\Utopia\Request;
+use Appwrite\Utopia\Response;
+use Utopia\App;
+use Utopia\Database\DateTime;
+use Utopia\Database\Document;
+use Utopia\Queue\Client;
+use Utopia\System\System;
+use Utopia\Validator\ArrayList;
+use Utopia\Validator\Text;
+
+App::init()
+    ->groups(['edge'])
+    ->inject('request')
+    ->action(function (Request $request) {
+
+        $token = $request->getHeader('authorization');
+        $token = str_replace(["Bearer"," "], "", $token);
+        $jwt = new JWT(System::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 600, 10);
+        try {
+            $payload = $jwt->decode($token);
+        } catch (JWTException $error) {
+            throw new Exception(Exception::USER_JWT_INVALID, 'Failed to verify JWT. ' . $error->getMessage());
+        }
+    });
+
+App::post('/v1/edge/sync')
```

Comment:
Maybe edge is not the best term here. Lets try `/v1/region` for region metadata (name, etc), and `/v1/region/syncs` unless we find a better name for reporting all the keys to sync.

Reply:
renamed to region